.github/workflows/publish.yml

```diff
@@ -93,6 +93,4 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          platforms:
-            - linux/amd64
-            - linux/arm64
+          platforms: linux/amd64,linux/arm64
```

Comment:
According to https://github.com/mozilla/remote-settings/pull/762

This is not enough